src/components/login/container/LoginDetails.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../styles/index.css';
+import newVideo from '../styles/video/newInstructions.mp4';
+
+const LoginDetails = () => (
+  <div className="login-details">
```

Comment:
You might want to wrap the component in a bootstrap `container class` so the page doesn't get distorted on tablets.

Reply:
I have done that, thanks